asapdiscovery-docking/asapdiscovery/docking/workflows/large_scale_docking.py

```diff
@@ -326,9 +352,12 @@ def large_scale_docking(inputs: LargeScaleDockingInputs):
     n_prepped_complexes = len(prepped_complexes)
     logger.info(f"Prepped {n_prepped_complexes} complexes")
 
-    if inputs.gen_du_cache:
-        logger.info(f"Generating DU cache at {inputs.gen_du_cache}")
-        prepper.cache(prepped_complexes, inputs.gen_du_cache)
+    if inputs.gen_cache:
```

Comment:
ok yeah I definitely could be missing something, but it seems like `cache_dir` and `gen_cache` should be unified into one directory. as far as I can tell, we're passing one as a cache directory for the `ProteinPrepper` step, and then also saving the same cached files to a different directory?

Reply:
I will make an issue for improving the caching to be more cache-like, but lets move on with this MVP and go from there.